src/fmt.jl

```diff
@@ -200,3 +222,20 @@ function fmt(x, syms::Symbol...; kwargs...)
     d = _add_kwargs_from_symbols(kwargs, syms...)
     fmt(x; d...)
 end
+
+# --------------------------------------------------------------------------------------------------
+
+# seed it with some basic default formatters
+for (t, c) in [(Integer,'d'),
+               (AbstractFloat,'f'),
+               (AbstractChar,'c'),
+               (AbstractString,'s'),
+               (ComplexInteger,'d'),
```

Comment:
For now, please remove the separate `ComplexInteger`, `ComplexFloat`, `ComplexRational` default specs.